.github/workflows/sonar_checks.yml

```
@@ -124,7 +124,7 @@ jobs:
           PR_NUMBER: ${{ env.PR_NUMBER }}
 
       - name: SonarCloud Scan
-        if: steps.pr_check.outputs.should_scan == 'true'
+        if: steps.pr_check.outputs.should_scan == 'true' && secrets.CICD_ORG_SONAR_TOKEN_CICD_BOT != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)